base/random.jl

```diff
@@ -1384,7 +1406,7 @@ function randexp_unlikely(rng, idx, x)
 end
 
 """
-    randn!([rng=GLOBAL_RNG], A::AbstractArray) -> A
+    randn!([rng::AbstractRNG=GLOBAL_RNG,] A::AbstractArray) -> A
```

Comment:
I thought putting it in the brackets makes more sense because otherwise it looks like you'd call this as `randn!(, A::AbstractArray)`, which you wouldn't.